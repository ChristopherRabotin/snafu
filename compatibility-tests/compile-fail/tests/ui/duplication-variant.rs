use snafu::prelude::*;

#[derive(Debug, Snafu)]
enum EnumError {
    #[snafu(display("an error variant"))]
    #[snafu(display("should not allow duplicate display"))]
    AVariant,
}

fn main() {}
