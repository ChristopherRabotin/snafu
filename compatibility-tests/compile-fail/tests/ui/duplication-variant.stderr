error: Multiple `display` attributes are not supported on an enum variant
 --> tests/ui/duplication-variant.rs:6:13
  |
6 |     #[snafu(display("should not allow duplicate display"))]
  |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
